### PR TITLE
workflows: Add more supporting code and bugfixes

### PR DESCRIPTION
**Pull request by bduffany**
- Return WebhookURL in the `Workflow` response so that the user can view it later via the workflows list view
- Fix bug that workflow ID isn't returned in `GetWorkflowsResponse`
- Fix auth flow bug (authenticatedUser.groupID doesn't necessarily match the group ID in the request context)
- Modify app routes to point to workflows
- Add SVG to image dir

---

**Version bump**: Minor

### CODE REVIEW

File: server/workflow/workflow_test.go [hunk 1]
```diff
@@ -159,4 +160,14 @@ func TestList(t *testing.T) {
 	rsp, err = bbClient.GetWorkflows(ctx2, req)
 	assert.Nil(t, err)
 	assert.Equal(t, 1, len(rsp.GetWorkflow()), "One workflow owned by USER2 should be returned")
+
+	if len(rsp.GetWorkflow()) == 1 {
```
Comment (tylerwilliams) [inline server/workflow/workflow_test.go:164]: remove if? since you assert it above?

**Review by tylerwilliams — APPROVED** [review]
LGTM (minor nits only)

Thanks for cleaning this up!

File: server/workflow/workflow_test.go (hunk 1 above)
Comment (bduffany) [reply server/workflow/workflow_test.go:164]: `assert` isn't fatal, it just adds to a list of successes / failures that are reported at the end of the test.

There's `require` which is fatal but I'd prefer using `assert` whenever possible

File: server/libmain/libmain.go [hunk 2]
```diff
@@ -66,6 +66,19 @@ var (
 
 	staticDirectory = flag.String("static_directory", "/static", "the directory containing static files to host")
 	appDirectory    = flag.String("app_directory", "/app", "the directory containing app binary files to host")
+
+	appRoutes = []string{
```
[the remark this answers is not in the document]
Comment (bduffany) [reply server/libmain/libmain.go:71]: Done

**Pull request merged**